pkgs/misc/emulators/caprice32/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, libpng, pkgconfig, SDL, freetype, zlib, mesa }:
+
+stdenv.mkDerivation rec {
+
+  pname = "caprice32";
+  version = "02102017";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ColinPitrat";
+    repo  = "caprice32";
+    rev = "53de69543300f81af85df32cbd21bb5c68cab61e";
+    sha256 = "12yv56blm49qmshpk4mgc802bs51wv2ra87hmcbf2wxma39c45fy";
+  };
+
+  patchPhase = "substituteInPlace cap32.cfg --replace /usr/local $out";
```

Comment:
Suggest using prePatch or postPatch instead. Replacing patchPhase prevents addition of patches while overriding like `caprice32.overrideAttrs (o: {patches = [./foo.patch];})`